cytominer_eval/tests/test_operations/test_hitk.py

```diff
@@ -0,0 +1,88 @@
+import os
```

Comment:
Please make sure to test `hitk` also _through the `evaluate()`_ function.

E.g. see [here](https://github.com/cytomining/cytominer-eval/blob/master/cytominer_eval/tests/test_evaluate.py). We test not only each individual eval function, but also using each through `evaluate.py`.

Reply:
Did this in a very small format